nsql/README.md

```diff
@@ -0,0 +1,100 @@
++------------+
+| trip_count |
++------------+
+| 199155     |
++------------+
+
+Time: 1.006369 seconds. 1 rows.
+```
+
+**Step 4.** Check the underlying query
+```shell
+spice sql 
+```
+
+Then:
+```shell
+sql> SELECT start_time, parent_span_id, span_id, task, substr(input, 0, 64) AS input, execution_duration_ms FROM runtime.task_history WHERE trace_id=(SELECT trace_id FROM runtime.task_history WHERE task='nsql') ORDER BY start_time ASC;
++----------------------------+------------------+------------------+---------------+-----------------------------------------------------------------+-----------------------+
+| start_time                 | parent_span_id   | span_id          | task          | input                                                           | execution_duration_ms |
++----------------------------+------------------+------------------+---------------+-----------------------------------------------------------------+-----------------------+
+| 2024-10-14T10:18:30.618720 |                  | 3ba34e0b690baf70 | nsql          | How many trips had more than two passengers?                    | 1005.1840000000001    |
+| 2024-10-14T10:18:30.621154 | 3ba34e0b690baf70 | c8a36151be8719a8 | ai_completion | {"messages":[{"role":"system","content":"```SQL\nCREATE TABLE I | 996.7589999999999     |
+| 2024-10-14T10:18:31.618123 | 3ba34e0b690baf70 | c8fd047ceaf7bc6a | sql_query     | SELECT COUNT(*) AS trip_count                                   | 5.555                 |
+|                            |                  |                  |               | FROM taxi_trips                                                 |                       |
+|                            |                  |                  |               | WHERE passenger                                                 |                       |
++----------------------------+------------------+------------------+---------------+-----------------------------------------------------------------+-----------------------+
+
+Time: 0.01162825 seconds. 3 rows.
+```
+
+
+### (Optional) Use a Local Model 
+**Step 1.** In the `spicepod.yaml`, uncomment the model `local`.
+
+**Step 2.** Restart the `spiced` server.
+
+**Step 3.** Run the NSQL tool, and select the local model.
+```shell
+>>> ~/.spice/bin/spice nsql
+Welcome to the Spice.ai NSQL REPL!
+Use the arrow keys to navigate: ↓ ↑ → ←
+? Select model:
+    oai
+  ▸ local
+```
+
+**Step 4.** Ask a question
+```shell
+nsql> What’s the highest tip any passenger gave?
++--------------------+
+| highest_tip_amount |
++--------------------+
+| 428.0              |
++--------------------+
+
+Time: 9.141290 seconds. 1 rows.
+```
+
+Step 5.** (Optional) Check the underlying query
+```shell
+sql> select start_time, parent_span_id, span_id, task, substr(input, 0, 64) as input, execution_duration_ms from runtime.task_history where trace_id=(select trace_id from runtime.task_history where task='nsql') order by start_time asc;
```

Comment:
Same suggestion as above. Split the code fence in two, one for SQL one for the results.